pkgs/development/compilers/dotnet/update.sh

```diff
@@ -265,11 +284,12 @@ update() {
     fi
     local major_minor_underscore=${major_minor/./_}
 
-    local release_content aspnetcore_version runtime_version
+    local release_content aspnetcore_version windowsdesktop_version runtime_version
     local -a sdk_versions
 
     release_content=$(release "$content" "$major_minor_patch")
     aspnetcore_version=$(jq -r '."aspnetcore-runtime".version' <<< "$release_content")
+    windowsdesktop_version=$(jq -r '."windowsdesktop".version' <<< "$release_content")
```

Comment:
Need to add this to the check that says whether it's up to date or not, because I think a desktop version can be released without the SDK changing.

Reply:
I don't think a .NET release can be changed after it was published? So as long as we are checking the release verson, we should be fine.
But looks like actually we aren't. The update script compares `$major_minor_patch` (which comes from `release-version`/`latest-release`) to `runtime_${major_minor_underscore}.version`, and technically the release version can differ from the runtime version.

Reply:
Never mind, they do straight up update existing releases and even change the date on them: https://github.com/dotnet/core/commit/a1400df8e678789e5295fb213366c6e1bec2a6f4#diff-72ae7001824195831a84fc7729d796c6681825b1ddaf75bea115095deb098f4a
So I guess we should version check all components, including aspnetcore, runtime and all sdk bands.

Reply:
Yeah, ideally we should check every single version, but since your PR is only about the windows part so you can add only that part if you want.